src/main/java/dev/openfga/sdk/api/OpenFgaApi.java

```diff
@@ -264,11 +300,24 @@ private CompletableFuture<ApiResponse<ExpandResponse>> expand(
 
         String path = "/stores/{store_id}/expand".replace("{store_id}", ApiClient.urlEncode(storeId.toString()));
 
+        Map<Attribute, String> telemetryAttributes = new HashMap<Attribute, String>() {
+            {
+                put(Attributes.FGA_CLIENT_REQUEST_METHOD, "Expand");
+                put(Attributes.FGA_CLIENT_REQUEST_STORE_ID, storeId);
+            }
+        };
+
+        try {
+            if (!isNullOrWhitespace(body.getAuthorizationModelId())) {
```

Comment:
How are you doing these custom changes in SDK Generator? A patch file?

Is it possible to avoid that?